jobbergate-composed/etc/slurm.conf

```diff
@@ -86,6 +86,8 @@ AccountingStorageHost=slurmdbd
 #
 # COMPUTE NODES
 NodeName=c[1-2] RealMemory=1000 State=UNKNOWN CPUs=4
+NodeName=jg-agent RealMemory=1000 State=UNKNOWN CPUs=1
```

Comment:
We can revert this file

Reply:
solved in [d6f53d1](https://github.com/omnivector-solutions/jobbergate/pull/513/commits/d6f53d18e0245bdccee891ca55fd4087fdee525e)